Add a redirect for the sponsorship package

jacobdeery opened the pull request:
waterloorocketry.com/sponsorship will now redirect to the sponsorship package pdf.

jacobdeery committed:
Add a redirect for the sponsorship package

waterloorocketry.com/package will now redirect to the sponsorship
package pdf.

akmorrison reviewed (approved):
I'd have used `sponsorship` as the redirect URL, solely out of a puerile sense of humor, but this works too

jacobdeery committed:
Rename package.html to sponsorship.html

Also, remove the useless meta charset tag as a driveby

akmorrison commented:
functionally, it works on my local machine. Putting `file:///home/aaron/rocketry/website/package.html` does the redirect in the way we hope the redirect would work

jacobdeery committed:
Update sponsors.html to use the new redirect

If we ever decide to change the PDF for the sponsorship package or make
it a video or something, we can just change it in sponsorship.html.

JaredWatson reviewed (approved):
Nice quick way to clean up the outward facing website. Lgtm
